jbehave-support-core/src/main/java/org/jbehavesupport/core/ws/WebServiceHandler.java

```diff
@@ -63,6 +80,8 @@
     private static final String CONTEXT_SEPARATOR = ".";
     private static final String REQUEST_POSTFIX = "Request";
     public static final String BRACKET_REGEX = "(.*)\\[(\\d+)\\](.*)";
+    public static final String SLASH_PREFIX = "/";
```

Comment:
Minor, but since its not in internal - do we want to have this as public?

Reply:
Let it be private )